v2/pkg/engine/datasource/graphql_datasource/graphql_tws_handler_test.go

```diff
@@ -240,96 +238,6 @@ func TestWebsocketSubscriptionClientError_GQLTWS(t *testing.T) {
 	}, time.Second, time.Millisecond*10, "server did not close")
 }
 
-func TestWebSocketSubscriptionClientInitIncludePing_GQLTWS(t *testing.T) {
-	if flags.IsWindows {
```

Comment:
Our previous implementation was Windows compatible. That should be our goal with this implementation too.

Reply:
It's compatible now